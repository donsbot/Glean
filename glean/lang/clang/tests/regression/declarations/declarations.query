query: |
    R where cxx1.DeclarationSrcRange { _, R }
transform: [gensort, []]
