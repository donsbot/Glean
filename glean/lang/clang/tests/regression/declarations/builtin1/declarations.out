[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 1,
      "lineEnd": 9,
      "columnEnd": 20
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 1,
      "lineEnd": 14,
      "columnEnd": 1
    }
  }
]